src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData-CSharp.Web/Program.Aspire.cs

```diff
@@ -27,7 +27,16 @@
 #else // IsAzureOpenAI || IsOpenAI || IsGHModels
 var openai = builder.AddAzureOpenAIClient("openai");
 openai.AddChatClient("gpt-4o-mini")
-    .UseFunctionInvocation();
+    .UseFunctionInvocation()
+    .UseOpenTelemetry(null, null, configure =>
+    {
+        // In development mode, enable the inclusion of sensitive data like
+        // prompt and user messages
+        if (builder.Environment.IsDevelopment())
+        {
+            configure.EnableSensitiveData = true;
+        }
+    });
```

Comment:
It's a long Program.cs already so if we can shorten this that would be great, e.g.:

```suggestion
    .UseFunctionInvocation()
    .UseOpenTelemetry(null, null, configure =>
        configure.EnableSensitiveData = builder.Environment.IsDevelopment());
```